aspnetcore/data/ef-mvc/intro.md

```diff
@@ -63,16 +63,16 @@ Open Visual Studio 2015 and create a new ASP.NET Core C# web project named "Cont
 
 * Wait for the **New ASP.NET Core Web Application (.NET Core)** dialog to appear
 
-* Select the **Web Application** template and ensure that **Authentication** is set to **Individual User Accounts**.
+* Select **ASP.NET 1.1** and the **Web Application** template.
 
-* Clear the **Host in the cloud** check box.
+* Set **Authentication** to **Individual User Accounts**.
 
 * Click **OK**
 
   ![New ASP.NET Project dialog](intro/_static/new-aspnet.png)
 
 > [!NOTE]
-> Don't miss setting authentication to **Individual User Accounts**. You won't be using authentication in this tutorial, but you need to enable it because of a limitation of .NET Core Preview 2 Visual Studio tooling. Scaffolding for MVC controllers and views only works when **Individual User Accounts** authentication is enabled.
+> Make sure the **Authentication** setting is **Individual User Accounts**. You won't be using authentication in this tutorial, but this setting automates some set-up tasks that you'd have to do manually otherwise.
```

Comment:
Yes scaffolding works without auth, and we can update the tutorial to not use auth later.